lib/platform/git/storage.ts

```diff
@@ -498,7 +506,10 @@ export class Storage {
         }
       }
       // istanbul ignore if
-      if (fileNames.length === 1 && fileNames[0] === 'renovate.json') {
+      if (
+        fileNames.length === 1 &&
+        (fileNames[0] === 'renovate.json' || fileNames[0] === '.gitmodules')
+      ) {
         fileNames.unshift('-f');
```

Comment:
I actually forget why this was originally added. Why is it also necessary for `.gitmodules`?

Reply:
I added it from an earlier version of this change, and I should be able to remove it. Should I remove the check for `renovate.json` as well?

Reply:
I don’t know.. leave it for now

Reply:
I think `.gitmodules` was added for git submodule support.